### ISSUE TITLE
mysql_common has breaking changes on version 0.30.1

### ISSUE THREAD

**Issue by altmannmarcelo** [readysettech/readyset#663]
**Describe the problem**
Module rust_mysql_common introduced some breaking changes on version 0.30.1 - https://github.com/blackbeam/rust_mysql_common/releases/tag/v0.30.1

This blocks readyset from seamlessly upgrading the module from version 0.29.2 to the latest

**To Reproduce**
upgrade any Cargo.toml to use `mysql_common = "0.30.6"

```
error[E0432]: unresolved import `mysql_common::value::convert::ConvIr`
  --> mysql-time/src/lib.rs:10:36
   |
10 | use mysql_common::value::convert::{ConvIr, FromValue, FromValueError};
   |                                    ^^^^^^ no `ConvIr` in `value::convert`
```

**Comment by altmannmarcelo** [comment]
@glittershark Nice. I am implementing support for Binlog Transaction Compression (#634).
Upstream has accepted the changes to handle [TRANSACTION_PAYLOAD_EVENT](https://github.com/blackbeam/rust_mysql_common/commit/22f0a559aea133c6f63ef5709e3d1a494243e36c) and the mysql_common module will need to be upgraded to latest version once it gets released. 
For now, I am cherry-picking the commit on top of v0.29.2.
If you happen to remember the WIP branch or PR let me know and I can have a look/take over.

**Comment by altmannmarcelo** [comment]
Fixed as part of 2e7c0c36a48cb64aef15c5c3d432dd178690b45b
